docs/update_release_notes.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Script to help with the SMQTK release process. Performs the following steps:
+#   - Poetry version (major, minor, or patch)
+#   - Rename release_notes/pending_release file to release_notes/version
+#   - Add reference to new release notes file in release_notes.rst
+#   - Add new release notes stub file
+#
+# Two git commits are created. One for the version bump and one for the new
+# release notes stub file.
+#
+
```

Comment:
We should add the line `set -e` at the beginning here. This will cause the script to exit if any in-situ error occurs instead of continuing along as if the error did not happen. Realization of the "error early and often" philosophy.